relayer-cli/src/commands/tx/packet.rs

```diff
@@ -22,6 +24,26 @@ pub struct TxRawPacketRecvCmd {
 
     #[options(free, required, help = "identifier of the source channel")]
     src_channel_id: ChannelId,
+
+    #[options(
+        help = "use the given signing key (default: `key_name` config)",
+        short = "k"
+    )]
+    key: Option<String>,
+}
+
+impl Override<Config> for TxRawPacketRecvCmd {
+    fn override_config(&self, mut config: Config) -> Result<Config, abscissa_core::FrameworkError> {
+        let src_chain_config = config.find_chain_mut(&self.src_chain_id).ok_or_else(|| {
+            FrameworkErrorKind::ComponentError.context("missing src chain configuration")
+        })?;
+
+        if let Some(ref key_name) = self.key {
+            src_chain_config.key_name = key_name.to_string();
+        }
+
+        Ok(config)
+    }
```

Comment:
maybe we undo these changes? i only used this for testing and we can always use a different config for the CLI

Reply:
Good idea!